cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -1397,3 +1469,29 @@ func CleanSriov() {
 	}
 	sriov.WaitStable(sriovclient)
 }
+
+func getDeviceMac(pod *corev1.Pod, device string) (string, error) {
+	getMacOutput, err := pods.ExecCommand(client.Client, *pod, []string{"ip", "link", "show", "dev", device})
+	if err != nil {
+		return "", err
+	}
+	macLine := strings.Trim(strings.Split(getMacOutput.String(), "\n")[1], " ")
+	return strings.Split(strings.Trim(macLine, " "), " ")[1], nil
```

Comment:
please check the len here before you do the `[1]` to be sure we are not going to fly